src/grid/grid.directive.ts

```diff
@@ -10,13 +10,18 @@ import { Directive, HostBinding, Input, OnInit } from "@angular/core";
 })
 export class GridDirective {
 	@HostBinding("class.bx--grid") baseClass = true;
+	@HostBinding("class.bx--grid--condensed") @Input() condensed = false;
 }
 
 @Directive({
 	selector: "[ibmRow]"
 })
 export class RowDirective {
 	@HostBinding("class.bx--row") baseClass = true;
+	@HostBinding("class.bx--row--condensed") @Input() condensed = false;
+	@HostBinding("class.bx--no-gutter") @Input() noGutter = false;
+	@HostBinding("class.bx--no-gutter--left") @Input() noLeftGutter = false;
+	@HostBinding("class.bx--no-gutter--right") @Input() noRightGutter = false;
```

Comment:
I wonder if instead of `noGutter` and friends, we flipped the naming and went with `gutter`/`leftGutter`/`rightGutter` ... It's a little weird from the implementation side, since we'd have to use a setter/OnChanges to flip the values, but maybe it makes more sense to write? Basically:

```html
<!-- disabling a feature -->
<div ibmRow [gutter]="false"></div>
```

vs.

```html
<!-- enabling the disablement of a feature? -->
<div ibmRow [noGutter]="true"></div>
```